src/ClassicUO.Client/LegionScripting/docs/API.py

```
@@ -1579,6 +1580,55 @@ def GetMultisInArea(x1: int, y1: int, x2: int, y2: int) -> list[Any]:
+def RemoveFriend(serial: int) -> bool:
+    """
+     Remove a mobile from the friends list by serial number.
+     Example:
+     ```py
+     API.RemoveFriend(0x12345)
+     ```
+    
+    """
+    pass
+
+def GetAllFriends() -> Any:
```

Ruff (0.13.1)

Undefined name `Any`

(F821)